frontend/src/components/contactsList.tsx

```diff
@@ -10,8 +18,9 @@ export interface Contact {
   notes?: string
   source?: string
   ownerID: string
-  relationships?: ContactRelationship[]
+  relationships: ContactRelationship[]
   importantDateEvent?: ImportantDateEvent[]
+  isFavorite?: boolean
```

Comment:
I think you need to create this isFavorite column in the database as well

Reply:
Also by default, what state would this be in? True or False?

Reply:
Added the column to the Contacts table in the database and set the default to False.